src/ert/job_queue/job_queue_node.py

```diff
@@ -196,6 +219,62 @@ def run_timeout_callback(self) -> None:
         if self.callback_timeout:
             self.callback_timeout(*self.callback_arguments)
 
+    # this function only works on systems where multiprocessing.Process uses forking
+    def run_done_callback_forking(self) -> Tuple[LoadStatus, str]:
```

Comment:
I like this part a lot! :tada: very nice!

Reply:
all honor and glory goes to @JHolba, and his house